pkg/cmd/server/api/types.go

```diff
@@ -40,6 +49,10 @@ type NodeConfig struct {
 	// PodManifestConfig holds the configuration for enabling the Kubelet to
 	// create pods based from a manifest file(s) placed locally on the node
 	PodManifestConfig *PodManifestConfig
+
+	// DockerExecHandlerName is the name of the handler to use for executing
+	// commands in Docker containers.
+	DockerExecHandlerName DockerExecHandlerType
```

Comment:
> Can that wait till later?

Sorry, config lives forever.  We don't have stable and unstable levels like the API.  The longer we wait, the more conversions we have to keep around forever.

I'll take care of moving old values and their associated conversions, so your code can be conversion free: just a new `DockerConfig` struct containing `DockerExecHandlerName`.